src/app/widgets/widget-gauge-ng-radial/widget-gauge-ng-radial.component.ts

```diff
@@ -171,7 +173,7 @@ export class WidgetGaugeNgRadialComponent extends BaseWidgetComponent implements
     this.gaugeOptions.valueDec = this.widgetProperties.config.numDecimal !== undefined && this.widgetProperties.config.numDecimal !== null ? this.widgetProperties.config.numDecimal : 2;
     this.gaugeOptions.majorTicksInt = this.widgetProperties.config.numInt !== undefined && this.widgetProperties.config.numInt !== null ? this.widgetProperties.config.numInt : 1;
     this.gaugeOptions.majorTicksDec = this.widgetProperties.config.numDecimal !== undefined && this.widgetProperties.config.numDecimal !== null ? this.widgetProperties.config.numDecimal : 2;
-    this.gaugeOptions.highlightsWidth = 0;
+    this.gaugeOptions.highlightsWidth = this.widgetProperties.config.gauge.highlightsWidth !== undefined && this.widgetProperties.config.gauge.highlightsWidth !== null ? this.widgetProperties.config.gauge.highlightsWidth : 5;
```

Comment:
Reworking initWidget() will remove the need to test highlightsWidth for undefined or null.

Reply:
I think specifically adding `highlightsWidth` to `defaultConfig` makes this check unnecessary.  I tested that by manually editing the local storage config to remove `highlightsWidth` and reloading the page, and the result was that the default width was used.